modules/programs/neomutt-accounts.nix

```diff
@@ -32,5 +32,24 @@ with lib;
         Extra lines to add to the folder hook for this account.
       '';
     };
+
+    onlyVirtualMailbox = mkOption {
+      type = types.bool;
+      default = false;
+      example = true;
+      description = ''
+        Don't use the mailboxes option of neomutt. Only use virtual-mailboxes.
```

Comment:
this suggests we can have several ones, so maybe it should be `onlyVirtualMailboxes` ? also I wonder if one wouldnt want to customize the default query `notmuch://?query=tag:inbox` but then these virtual maiboxes could possibly be abstracted to be leveraged by alot too ?